resources/views/components/cd/form/form-item.blade.php

```diff
@@ -0,0 +1,21 @@
+@php
+    $is_required = $required ?? true;
```

Comment:
Will it work if $required is not specified? 
Also how do you see form-item component is used?

Reply:
Yes, the null coallessing operator is providing a default.  This PR is really just about making sure that the components can be integrated properly; the next update I have, which includes more components, will make the default false, as you had suggested.